test/wsdl-load-from-memory-test.js

```diff
@@ -39,6 +39,26 @@ describe('wsdl-load-from-memory-tests', function() {
         done()
       })
     });
+
+    it('should load a wsdl with no imports directly from memory and in a sync manner', function (done) {
```

Comment:
nitpick: can also name this `should load a wsdl synchronously with no imports directly from memory` or `should honour synchronous loading of a wsdl` but feel free to keep as is :-)

Reply:
will rename and push :-)